textAdventure.js

```diff
@@ -0,0 +1,229 @@
+class Room {
+    constructor(name,description,items,north,east,south,west){
+    this.name=name //string
+    this.description=description //string
+    this.items=items //array?
+    this.north=north //room object
+    this.south=south // 
+    this.east=east
+    this.west=west
```

Comment:
You can set default values for properties either by using the logical `or` operator `||` where you define the value `this.west = west || undefined`, or by setting a default in the constructor as you would with a normal function `constructor(argOne, argTwo = default)`